apps/language_server/lib/language_server/dialyzer.ex

```diff
@@ -442,7 +442,18 @@ defmodule ElixirLS.LanguageServer.Dialyzer do
 
   defp dialyxir_format({_, _, {warning_name, args}}) do
     %{^warning_name => warning_module} = Dialyxir.Warnings.warnings()
-    <<_::binary>> = apply(warning_module, :format_long, [args])
```

Comment:
Is there a reason to drop the `<<_::binary>>` check? I assume it's there to ensure that we're getting a binary back

Reply:
The way those functions work, that will always be true. It was simply superfluous. I can put it back though if you feel strongly!

Reply:
I see it more as a guard against it changing in the future, but I guess that's unlikely and I probably wouldn't be arguing for adding it if it wasn't already there ;)